framework/db/Query.php

```diff
@@ -844,6 +849,119 @@ public function addParams($params)
+     *
+     * @param array $tables array of tables in the format allowed for [[from()]]:
+     *
+     * - numeric key, string value == table without alias, or table with alias specified after the name, separated by white space.
+     * - string key, string value == table with alias.
+     * - string key, obect value == subquery with alias. This alias will not be populated.
+     * @see from()
+     * @see join()
+     * @see getAlias()
+     * @since 2.0.7
+     */
+    protected function populateAliases($tables)
+    {
+        foreach ($tables as $alias => $tableName) {
+            if (is_object($tableName)) {
+                continue;
+            }
+            if (is_string($alias)) {
+                $this->_aliases[$tableName] = $alias;
+            } elseif (preg_match('/^(\\{\\{(\w+)\\}\\}|(.*?))(\s+AS\s+|\s+)(\\{\\{\w+\\}\\}|\w+)$/i', $tableName, $matches)) {
+                $this->_aliases[$matches[2] ?: $matches[3]] = $matches[5];
+            }
+        }
+    }
+
+    /**
+     * Returns the alias of a table in this query.
+     *
+     * Aliases are available when a table alias has been specified by calling either [[from()]],
+     * [[join()]] or other join methods. If no alias has been defined, the original table name is
+     * returned without modification.
+     *
+     * Note, that if a table is used multiple times in the query (e.g. self-join situations)
+     * relying on this function may not give the expected result.
+     *
+     * Usage example:
+     *
+     * ```php
+     * $query = (new Query)->from(['u' => 'user']);
+     *
+     * echo $query->getAlias('user'); // "u"
+     * $query->andWhere($query->applyAlias('user', 'name') => 'cebe');
+     * // SELECT * FROM user u WHERE u.name = 'cebe';
+     * ```
+     *
+     * Note also, that using this method might not work in all cases because the alias may change
+     * later when the query is modified. An alternative Syntax for resolving table aliases is
+     * described in the [Guide about alias handling](db-querybuilder#aliases) and can be used as follows:
+     *
+     * ```php
+     * // {{@user}} will resolve to the table alias
+     * $query->where('{{@user}}.id = post.author_id')
+     * ```
+     *
+     * @param string $table the table name.
+     * @return string the table alias.
+     * @see applyAlias() for disambiguating columns.
+     * @since 2.0.7
+     */
+    public function getAlias($table)
+    {
+        if (isset($this->_aliases[$table])) {
+            return $this->_aliases[$table];
+        }
+        return $table;
+    }
+
+    /**
+     * Disambiguate a column name by looking up the table alias.
+     *
+     * This method will look up the alias for a table using [[getAlias()]] and
+     * return the disambiguated column name. e.g. for `applyAlias('user', 'name')`
+     * it will return `u.name` if the table alias was `u`.
+     *
+     * This function can be used to disambiguate column names in situations
```

Comment:
This function can be used → It can be used

Because you're calling it a method above.